index.html

```diff
@@ -30,6 +33,175 @@ <h2>I’am a Software Developer</h2>
+                </div>
+                <div class="button">See Project</div>
 
+                <!-- work two third container -->
+                <div class="worktwo">
+                    <h2>Profesional Art <br> Printing Data</h2>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <div class="stacktwo">
+                        <h3>css</h3>
+                        <h3>html</h3>
+                        <h3>Ruby</h3>
+                    </div>
+                </div>
+                <div class="button">See Project</div>
+
+                <!-- work two fouth container -->
+                <div class="worktwo">
+                    <h2>Profesional Art <br> Printing Data</h2>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <div class="stacktwo">
+                        <h3>css</h3>
+                        <h3>html</h3>
+                        <h3>Ruby</h3>
+                    </div>
+                </div>
+                <div class="button">See Project</div>
+                
+                <!-- work two fifth container -->
+                <div class="worktwo-container">
+                    <div class="worktwo">
+                        <h2>Profesional Art <br> Printing Data</h2>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                        <div class="stacktwo">
+                            <h3>css</h3>
+                            <h3>html</h3>
+                            <h3>Ruby</h3>
+                        </div>
+                    </div>
+                    <div class="button">See Project</div>
+            </div>
+
+            <!-- work two sixth container -->
+            <div class="worktwo-container">
+                <div class="worktwo">
+                    <h2>Profesional Art <br> Printing Data</h2>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                    <div class="stacktwo">
+                        <h3>css</h3>
+                        <h3>html</h3>
+                        <h3>Ruby</h3>
+                    </div>
+                </div>
+                <div class="button">See Project</div>
+        </div>
+    </section>
+
+    <!-- About Section -->
+    <section>
+        <div class="aboutme">
+            <h1>About me</h1>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <div class="about-button"><a href="#">Get My Resume</a></div>
+        </div>
```

Comment:
- [ ] Dear @chibunduonyeje1, Please use a `a` or `button` tag for your `Get My Resume` instead of a `div` tag. Remember in the future you are going to use this to link to the Resume also this will provide a semantic-wise advantage.